### PR TITLE
Update first sentence in README

### PR DESCRIPTION
**Pull request by fritzgoebel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
 [![DOI](https://joss.theoj.org/papers/10.21105/joss.02260/status.svg)](https://doi.org/10.21105/joss.02260)
 
 Ginkgo is a high-performance linear algebra library for manycore systems, with a
-focus on sparse solution of linear systems. It is implemented using modern C++
-(you will need at least C++14 compliant compiler to build it), with GPU kernels
+focus on the solution of sparse linear systems. It is implemented using modern C++
+(you will need at least a C++14 compliant compiler to build it), with GPU kernels
```
Comment (upsj) [inline README.md:18]:
```suggestion
(you will need an at least C++14 compliant compiler to build it), with GPU kernels
```

**Review by upsj — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -14,8 +14,8 @@
 [![DOI](https://joss.theoj.org/papers/10.21105/joss.02260/status.svg)](https://doi.org/10.21105/joss.02260)
 
 Ginkgo is a high-performance linear algebra library for manycore systems, with a
-focus on sparse solution of linear systems. It is implemented using modern C++
-(you will need at least C++14 compliant compiler to build it), with GPU kernels
+focus on the solution of sparse linear systems. It is implemented using modern C++
+(you will need an at least C++14 compliant compiler to build it), with GPU kernels
 implemented in CUDA and HIP.
```
Comment (yhmtsai) [inline README.md:19]:
```suggestion
implemented in CUDA, HIP, and oneAPI.
```

**Review by yhmtsai — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (upsj) [reply README.md:19]: maybe rather DPC++ or SYCL?

File: README.md [hunk 3]
```diff
@@ -14,9 +14,9 @@
 [![DOI](https://joss.theoj.org/papers/10.21105/joss.02260/status.svg)](https://doi.org/10.21105/joss.02260)
 
 Ginkgo is a high-performance linear algebra library for manycore systems, with a
-focus on sparse solution of linear systems. It is implemented using modern C++
-(you will need at least C++14 compliant compiler to build it), with GPU kernels
-implemented in CUDA and HIP.
+focus on the solution of sparse linear systems. It is implemented using modern C++
+(you will need an at least C++14 compliant compiler to build it), with GPU kernels
+implemented in CUDA, HIP, and oneAPI.
```
Comment (upsj) [inline README.md:19]:
```suggestion
implemented in CUDA, HIP, and DPC++.
```
oneAPI is the software framework, DPC++ is the programming model

**Pull request merged**